src/scss/_common.scss

```diff
@@ -207,3 +207,71 @@ body {
 .node-form .field--name-field-hero thead .field-label {
   display: none;
 }
+
+// Select2 to look and feel more Gin-like.
+.select2-container {
+  .select2-selection {
+    border: 1px solid var(--colorGinFormElementBorder);
+    border-radius: 6px;
+    box-shadow: 0 1px 2px rgba(0, 0, 0, .25);
```

Comment:
You can use variable inside the rgba as well so like ($color-black, .25).

Reply:
But this is admin theme so you can leave it if it doesn't bother you.

Reply:
TBH That's a copy-paste and I feel the shame. 😁